src/containers/share-sound-to-tiktok-modal/ShareSoundToTikTokModal.tsx

```diff
@@ -150,31 +151,37 @@ const ShareSoundToTikTokModal = () => {
     }
   }
 
-  const mobileProps: Partial<ModalProps> = {
-    anchor: Anchor.BOTTOM,
-    showDismissButton: false,
-    verticalAnchorOffset: MODAL_OFFSET_PIXELS
-  }
-
-  return (
+  return mobile ? (
+    <Drawer onClose={handleClose} isOpen={!IS_NATIVE_MOBILE && isOpen}>
```

Comment:
should we just gate this section on !IS_NATIVE_MOBILE rather than only gating isOpen on it?